### PR TITLE
Task contact

### PR DESCRIPTION
**Pull request by ethan-garrett**
Added contact section content and styled them to be fully responsive as well as replacing placeholder image in Intro Section with a gif

### CODE REVIEW

File: portfolio.html [hunk 1]
```diff
@@ -50,6 +50,22 @@ <h3>Who am I?</h3>
 	</section>
 	<h2 id="Contact">Contact</h2>
 	<footer class="contactSection">
+		<div class="contactSectionContainer">
+			<h3>Contact details</h3>
+			<p>The best way to contact me is through email, though I'm also on Linkedin and Github.</p>
+			<div>
+				<p>My Email is: <strong>garrettethan63@gmail.com</strong></p>
+				<img src="emailIcon.png" alt ="" class="contactHide">
```
Comment (luke-grayland) [inline portfolio.html:58]: Is this image icon supposed to disappear when I reduce my screen width? Also, how about having the images spread across horizontally when the screen is smaller to leave less empty space?

File: portfolio.html [hunk 2]
```diff
@@ -13,7 +13,7 @@
 	</nav>
 	<section class="introSection">
 		<div class="introImage">
-			<img src="tetris.jpg" alt="Man Coding">
+			<img src="tetris.gif" alt="Man Coding">
 		</div>
 		<div class="introTextContainer">
 			<p>Hey There</p>
```
Comment (luke-grayland) [inline portfolio.html:19]: Could the 'About Me', 'Portfolio', and 'Contact' page headers be centered with text-align: center?

Comment (luke-grayland) [inline portfolio.html:19]: Could you use camel case when naming the ID of portfolio (i.e. 'portfolio' rather than 'Portfolio') for consistency?

**Review by luke-grayland — COMMENTED** [review]
Line 103: Could you leave a space between your selector and the opening curly bracket for consistency?

File: portfolio.html (hunk 1 above)
Comment (ethan-garrett) [reply portfolio.html:58]: Email icon is supposed to disappear as it is not required so to save space on smaller screens it disappears

**Review by luke-grayland — APPROVED** [review]

**Pull request merged**